exporter/opensearchexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenSearch Exporter
+
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [devel]   |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
+
+This exporter supports sending OpenTelemetry signals as documents to [OpenSearch](https://www.opensearch.org).
+
+The documents are sent using [observability catalog](https://github.com/opensearch-project/opensearch-catalog/tree/main/schema/observability) schema.
+
+## Configuration options
+
+- `endpoints`: List of OpenSearch URLs. If endpoints is missing, the
+  OPENSEARCH_URL environment variable will be used.
+- `num_workers` (optional): Number of workers publishing bulk requests concurrently.
```

Comment:
Is there a default to note here?  Presumably `1` but it would be good to call out here.  It's also odd that this is noted as `optional` while other values have their default noted, which presumably also means they're optional.

Reply:
This value is passed to the OpenSearch Go client as well. It defaults to the number of CPUs.

Reply:
Would be good to link to the details about how the default is determined, then.  I presume it's actually `GOMAXPROCS` under the hood, which may be an important distinction for some users.